src/ScottPlot4/ScottPlot.Cookbook/Recipes/Plottable/VectorField.cs

```diff
@@ -143,4 +145,47 @@ public void ExecuteRecipe(Plot plt)
             vf.MarkerSize = 3;
         }
     }
+
+    public class VectorFieldArbitrary : IRecipe
+    {
+        public ICategory Category => new Categories.PlotTypes.VectorField();
+        public string ID => "vectorField_arbitrary";
+        public string Title => "Arbitrary Vectors";
+        public string Description => "Vectors can be placed arbitrarily in coordiante space " +
+            "(they are not required to be in a grid)";
+
+        public void ExecuteRecipe(Plot plt)
+        {
+            // Create a vector field list and add it to the plot
+            var field = plt.AddVectorFieldList();
```

Comment:
I think there should be (and if there is, we should show it off) an option to just put the vectors into the list at construction

Reply:
I mostly agree... I'll add it, but FYI here's my hesitation:

Since this is a List plot type, one advantage of not accepting data in the constructor is that it forces people to interact with the List in the plottable. It simplifies the documentation in that there's only one way to add arrows.

If people only know how to pass in arrows, it could be less obvious how to add more or modify existing ones.